DCO

```diff
@@ -0,0 +1,37 @@
+Developer Certificate of Origin
```

Comment:
@dockerymick No need to review this file, only the changes in the README.md file below.

This one is standard legal text copied from other repos.